inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/Constants.java

```diff
@@ -187,7 +190,18 @@ public final class Constants {
                     .noDefaultValue()
                     .withDescription(
                             "The format of multiple sink, it represents the real format of the raw binary data");
-
+    public static final ConfigOption<String> PATTERN_PARTITION_MAP =
+            ConfigOptions.key("pattern.partition.map")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "Pattern rules and partition maps");
```

Comment:
detailed descrption like a example for the parameter may be good for reading

Reply:
OK, I will fixed!